### ISSUE TITLE
Check connection and version of database

### ISSUE THREAD

**Issue by Romerski** [mevdschee/php-crud-api#274]
Hi again,

I need to do two things: check if connection is successful with sql server (without selecting any table) and get sql server version. I suppose I can't do it without modify api.php file. 

For version I need to execute "SELECT @@VERSION". There is no option to do raw query, right?

Could you tell me where can I add this functions on script?

Thanks!!

**Comment by mevdschee** [comment]
>Could you tell me where can I add this functions on script?

Search for the relevant "connect" call, e.g. here:

https://github.com/mevdschee/php-crud-api/blob/master/api.php#L617

I hope this helps.

**Comment by Romerski** [comment]
Thanks!! 

Any idea on how and where create a custom get call for example /api.php/get_version to execute custom raw query (select @@version)?

**Comment by mevdschee** [comment]
Did you look at php-sp-api?

https://github.com/mevdschee/php-sp-api

That is the easy way :-)

**Comment by mevdschee** [comment]
ah.. wait .. you were on sql server

**Comment by Romerski** [comment]
Yeah, unfortunately i'm on sql server :( Would be great to execute a procedure or a raw query to get sql version and also to check connection is working fine.

If you have any ideas on how to make it work would appreciate it. Thanks again!

**Comment by Romerski** [comment]
Well, for the moment I've implemented a "dirty" piece of code on executecommand() function to execute GET call on /check_connection request retrieving sql server version. 

So I think I can work with this momentarily.

I would also like to ask if you have tried to implement this library in codeigniter with success and how. I tried adding api.php as controller to access directly by http://localhost/api but it returns entity not found. And can't access directly through absolute URL, codeigniter doesn't let.

Thanks again for your time.
